wgpu-hal/src/metal/adapter.rs

```diff
@@ -850,20 +850,24 @@ impl super::PrivateCapabilities {
             } else {
                 Self::version_at_least(major, minor, 11, 0)
             },
+            //Depth clipping is supported on all macOS GPU families and iOS family 4 and later
+            supports_depth_clamping: device.supports_feature_set(MTLFeatureSet::iOS_GPUFamily4_v1)
+                        || os_is_mac
```

Comment:
just double-checking if `cargo fmt` agrees :)

Reply:
Unless I"m doing it wrong, it's a'ok :D